files/galaxy/tpv/tools.yml

```diff
@@ -886,3 +886,10 @@ tools:
 
   toolshed.g2.bx.psu.edu/repos/iuc/semibin/semibin/.*:
     mem: 8
+
+  toolshed.g2.bx.psu.edu/repos/galaxyp/msconvert/msconvert/.*:
+    params:
+      docker_run_extra_arguments: --user 999
+    scheduling:
+      require:
+        - docker
```

Comment:
Do we need this? It is [already specified in the TPV shared database](https://github.com/galaxyproject/tpv-shared-database/blob/fe3ba41340dac17de585738c6a39539ca776c8b1/tools.yml#L632-L634), and if it was removed from there, then I assume the error would go away anyways.

```suggestion
```

Reply:
Right, the config will be merged, right? In that case, I don't think we would need the `scheduling` section here only the `param` section is needed.